packages/sdk-utilities/src/lib/url.utils.test.ts

```diff
@@ -0,0 +1,85 @@
+/**
+ *
+ * Copyright © 2025 Ping Identity Corporation
+ *
+ **/
+
+import { getBaseUrl, parseQuery, resolve, stringify } from './url.utils.js';
+
+interface QueryString {
+  [key: string]: { foo?: string; goo?: string };
+}
+
+describe('The URL utility', () => {
+  const querystrings: QueryString = {
+    '': {},
+    'foo=bar': { foo: 'bar' },
+    'foo=bar&goo=baz': { foo: 'bar', goo: 'baz' },
+    'foo=b%20r': { foo: 'b r' },
+  };
+
+  it('correctly determines base URL', () => {
+    const tests = [
```

Comment:
@ryanbas21 @cerebrl any reason why there are duplicate tests here for each url?

Reply:
I'm honestly not sure. These tests have been around for 6 years. But, if we toss this utility function it's testing with a more native method, then we can just delete a lot of these tests.

Reply:
Oh, after looking at this one more time, I think I see what you're talking about here. If you're referring to the fact that some items in this array are duplicates, like `['http://domain.com', 'http://domain.com']` and/or `['https://domain.com', 'https://domain.com']`, if you look closely, the difference is the scheme, `http` versus `https`.

Reply:
Are you sure because lines 23 and 24 look exactly the same to me... regardless we can remove this test and opt for native methods

Reply:
I think it's the trailing slash removal he's testing? IDK, now I'm starting to think I'm missing something ... lol